source/solvers/vof.cc

```diff
@@ -221,6 +470,105 @@ VolumeOfFluid<dim>::assemble_system_rhs()
   this->system_rhs.compress(VectorOperation::add);
 }
 
+template <int dim>
+void
+VolumeOfFluid<dim>::assemble_system_rhs_dg()
+{
+  const DoFHandler<dim> *dof_handler_fd =
```

Comment:
```suggestion
  const DoFHandler<dim> *dof_handler_fluid =
```
or change to `dof_handler_fd` at other instances